training/general/src/environments/tensorflow-2.12-cuda11/tests/src/TF_bert.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
+from transformers import AutoTokenizer, TFAutoModelForSequenceClassification

```

Comment:
Can you follow the naming convention of the other files in the repo? Other files use lower case in the file name. Can you change the upper case file name, TF_bert.py, to lower case?

Reply:
sure! file name has been changed